### PR TITLE
Updated how-to-file-an-issue

### PR DESCRIPTION
**Pull request by KillMonger1**
Addresses #1

### CODE REVIEW

File: docs/contributing/how-to-file-an-issue.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ---
-title: How to File an Issue
+शीर्षक: एक समस्या कैसे दर्ज करें
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `शीर्षक` => `title`

this will be used to identify title. So we will not translate this

Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `समस्या` => `इशू`

File: docs/contributing/how-to-file-an-issue.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 ---
 
-The [issue tracker](https://github.com/gatsbyjs/gatsby/issues) is the preferred channel for bug reports, feature requests and [submitting pull requests](/contributing/how-to-open-a-pull-request/).
+[issue tracker](https://github.com/gatsbyjs/gatsby/issues) बग रिपोर्ट, सुविधा अनुरोध और [पुल रिक्वेस्ट सबमिट करना](/contributing/how-to-open-a-pull-request/) के लिए पसंदीदा चैनल है।
 
-If you want your issue to be resolved quickly, please include in your issue:
+यदि आप चाहते हैं कि आपका मुद्दा जल्दी से हल हो जाए, तो कृपया अपने मुद्दे में यह शामिल करें:
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `मुद्दा` => `इशू`

change at both places in this line

File: docs/contributing/how-to-file-an-issue.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
-- Gatsby version, Node.js version, OS version
-- The contents of your `gatsby-config.js` and `package.json` as well as your
-  `gatsby-node.js`, `gatsby-browser.js` `gatsby-ssr.js` files depending on
-  changes you've made there.
+- Gatsby वर्शन, Node.js वर्शन, OS वर्शन
+- आपके `gatsby-config.js` और` package.json` की कंटेंट्स के साथ-साथ आपके
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]:
```
और` package.json` की
```
=>
```
और `package.json` की
```

File: docs/contributing/how-to-file-an-issue.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+   `gatsby-node.js`,` gatsby-browser.js` `gatsby-ssr.js` फाइलें जो
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]:
```
   `gatsby-node.js`,` gatsby-browser.js` `gatsby-ssr.js` फाइलें जो
```
=>
```
 `gatsby-node.js`, `gatsby-browser.js` `gatsby-ssr.js` फाइलें जो
```

File: docs/contributing/how-to-file-an-issue.md [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+   आपके द्वारा वहां किए गए परिवर्तन पर निर्भर करती है ।
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `   आपके द्वारा वहां किए गए परिवर्तन पर निर्भर करती है ।` => `आपके द्वारा वहां किए गए परिवर्तन पर निर्भर करती है।`

File: docs/contributing/how-to-file-an-issue.md [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
 
-Please do not use the issue tracker for personal support requests. [Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) (**gatsby** tag) and the [Gatsby Discord](https://gatsby.dev/discord) are better places to get help.
+कृपया व्यक्तिगत सपोर्ट अनुरोधों के लिए समस्या ट्रैकर का उपयोग न करें। [Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) (**gatsby** tag) and the [Gatsby Discord](https://gatsby.dev/discord) are better places to get help.
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]:
```
[Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) (**gatsby** tag) and the [Gatsby Discord](https://gatsby.dev/discord) are better places to get help.
```
=>
```
[Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) (**gatsby** टैग) और [Gatsby Discord](https://gatsby.dev/discord) मदद लेने के लिए सही जगह है।
```

File: docs/contributing/how-to-file-an-issue.md [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
 
-### Special Note on Issues
+### इन मुद्दों पर विशेष ध्यान दें
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `मुद्दों` => `इश्यूज`

File: docs/contributing/how-to-file-an-issue.md [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
 
-If an issue is affecting you, start at the top of this list and complete as many tasks on the list as you can:
+यदि कोई समस्या आपको प्रभावित कर रही है, तो इस सूची के शीर्ष पर शुरू करें और सूची के कई कार्यों को पूरा करें:
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `समस्या` => `इशू`

File: docs/contributing/how-to-file-an-issue.md [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
 
-1.  If there is an issue, add a reaction or more details to the issue to indicate that it's affecting you
-2.  If there is an issue and you can add more detail, write a comment describing how the bug is affecting OR if you can, write up a work-around for the bug
-3.  If there _is not_ an issue, write the most complete description of what's happening, preferably with link to a Gatsby site that reproduces the problem or [create a reproducible test case](/contributing/how-to-make-a-reproducible-test-case/)
-4.  Offer to help fix the bug (and it is totally expected that you ask for help; open-source maintainers want to help contributors)
-5.  [Deliver a well-crafted, tested PR](/contributing/how-to-open-a-pull-request/)
+1.  यदि कोई समस्या है, तो इंगित करने के लिए प्रतिक्रिया या अधिक विवरण जोड़ें कि यह आपको प्रभावित कर रहा है
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `समस्या` => `इशू`

Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `जोड़ें` => `ऐड करे`

File: docs/contributing/how-to-file-an-issue.md [hunk 10]
```diff
@@ -1,24 +1,24 @@
...
+2. अगर कोई समस्या है और आप और अधिक विवरण जोड़ सकते हैं, तो एक टिप्पणी लिखें जिसमें लिखा गया है कि बग कैसे प्रभावित हो रहा है या यदि आप कर सकते हैं, तो बग के लिए काम के बारे में लिखें
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `समस्या` => `इशू`

Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `जोड़` => `ऐड कर`

File: docs/contributing/how-to-file-an-issue.md [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
+3. यदि कोई समस्या _नहीं है_, तो जो कुछ भी हो रहा है उसका सबसे पूरा विवरण लिखें, अधिमानतः एक गैट्सबी साइट के लिंक के साथ जो समस्या को पुन: उत्पन्न करता है या [एक प्रतिलिपि प्रस्तुत करने योग्य टेस्ट केस बनाएँ](/contributing/how-to-make-a-reproducible-test-case/)
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `समस्या` => `इशू`

Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `गैट्सबी` => `Gatsby`

File: docs/contributing/how-to-file-an-issue.md [hunk 12]
```diff
@@ -1,24 +1,24 @@
...
+4. बग को ठीक करने में मदद करने के लिए प्रस्ताव (और यह पूरी तरह से अपेक्षित है कि आप मदद मांगें; ओपन-सोर्स मेंटेनर योगदानकर्ताओं की मदद करना चाहते हैं)
+5.  [एक अच्छी तरह से तैयार व परीक्षण किया  हुआ  PR बनाये ](/contributing/how-to-open-a-pull-request/)
```
Comment (arshadkazmi42) [inline docs/contributing/how-to-file-an-issue.md]: `परीक्षण किया  हुआ  PR बनाये` => `परीक्षण किया हुआ PR बनाये`

fix extra spaces

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Good word.
I added some reviews. It will be good to go post that.
You can read about our review process here [Review Process](https://github.com/gatsbyjs/gatsby-hi/blob/master/contibuting.md#%E0%A4%B0%E0%A4%BF%E0%A4%B5%E0%A5%8D%E0%A4%AF%E0%A5%81-%E0%A4%AA%E0%A5%8D%E0%A4%B0%E0%A5%8B%E0%A4%B8%E0%A5%87%E0%A4%B8)

**Comment by KillMonger1** [comment]
I've made all changes

File: docs/contributing/how-to-file-an-issue.md (hunk 2 above)
Comment (arshadkazmi42) [reply docs/contributing/how-to-file-an-issue.md]: `आपका` => `आपके`

sorry about this. I missed this.

**Review by arshadkazmi42 — APPROVED** [review]
Great work.
Thank you for working on this.

File: docs/contributing/how-to-file-an-issue.md (hunk 2 above)
Comment (arshadkazmi42) [reply docs/contributing/how-to-file-an-issue.md]: ignore this. I made this fix.

**Pull request merged**